frontend/src/components/FilterButtons/FilterButtons.js

```diff
@@ -0,0 +1,108 @@
+import styles from "./FilterButtons.module.css";
+import { useState } from "react";
+import api from "../api";
```

Comment:
Since you changed where FilterButtons is located, you should update the import from api.js.

Reply:
"@/utils/api"